p/themes/icons/bookmark-add.svg

```diff
@@ -1,6 +1,6 @@
-<svg xmlns="http://www.w3.org/2000/svg" height="16" width="16">
-<g transform="translate(-141.0002,-807)" fill="#bebebe">
-<path d="m143,807,0,13,4-4,4,4,0-4,0-1-2,0,0-4,2,0,0-4z"/>
-<path d="m152,810,0,2-2,0,0,2,2,0,0,2,2,0,0-2,2,0,0-2-2,0,0-2-2,0z"/>
-</g>
+<svg xmlns="http://www.w3.org/2000/svg" width="16" height="16">
+	<g fill="#666">
```

Comment:
The indentation's very much optional to me though, unless it were really complex maybe. :-)

(If you and/or @Alkarex prefer it this way it's certainly fine by me.)